packages/icon-button-toggle/src/icon-button-toggle-base.ts

```diff
@@ -55,6 +55,13 @@ export class IconButtonToggleBase extends BaseElement {
   }
 
   protected handleClick() {
+    const onIconSlotted = this.querySelector('[slot="onIcon"]');
```

Comment:
Shouldn't this be `this.shadowRoot.querySelector`?

Reply:
this is purposefully seeking the light dom's projected elements (not the default icon). Thought it would be cleaner than selecting the slot and using `assignedNodes`. (I also wonder what the performance indications are for `assignedNodes`)